dashboard/db/migrate/20200928223748_create_resources.rb

```diff
@@ -0,0 +1,12 @@
+class CreateResources < ActiveRecord::Migration[5.0]
+  def change
+    create_table :resources do |t|
+      t.string :name
+      t.string :url
+      t.string :embed_slug
```

Comment:
what do folks think about naming this `key` or `slug`? I'm not sure the "embed" part helps me understand what it is. `url_slug` could be another option, though then that might make me want to rename `url` to `target_url` or something like that. Thoughts @dmcavoy ?

Reply:
I think I was reviewing at the same time as you Dave. Sorry Bethany if that is confusing! I suggested changing name to title and moving it to properties. And changing embed_slug to key. So I think we are in agreement.

Reply:
`key` sounds good to me!